subsys/bluetooth/audio/aics.c

```diff
@@ -67,34 +67,39 @@ static ssize_t read_description(struct bt_conn *conn,
 	BT_GATT_SECONDARY_SERVICE(BT_UUID_AICS),                               \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_STATE,                             \
 			       BT_GATT_CHRC_READ | BT_GATT_CHRC_NOTIFY,        \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY,   \
 			       read_aics_state, NULL, &_aics),                 \
 	BT_GATT_CCC(aics_state_cfg_changed,                                    \
-		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE_ENCRYPT),           \
+		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE_ENCRYPT                 \
+			 | BT_GATT_PERM_READ_128B_ENTROPY | BT_GATT_PERM_WRITE_128B_ENTROPY) \
```

Comment:
```suggestion
			 | BT_GATT_PERM_WRITE_128B_ENTROPY) \
```

We don't need that sort of requirement for reading the CCCD

Reply:
Why is that though? As it does have read permission, why does the read not fall under the general security requirements?

Reply:
Yes, you are right, Ill update it